README.md

```diff
@@ -213,7 +213,12 @@ Users who have access to 1-1 chat also have a profile on Crisp that reflects dat
 
 ### Reporting
 
-<!-- TODO -->
+Google Data Studio is an online tool for converting data into customizable informative reports and dashboards.
+
+The reports are generated by writing custom sql queries that return actionable data to Data Studio. Filters are applied in Data Studio allowing
+data to be segregated into different time periods based on the data createdAt date
+
+Reports are private and are only viewed by the Bloom admin.
```

Comment:
this last line is throwing me a bit, who is a bloom admin? might be best to leave that last line out